pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -202,6 +202,7 @@ stdenv.mkDerivation rec {
             # about why to create a subdir for the workspace.
             cp -r ${workspaceDir} wd && chmod u+w wd && cd wd
 
+            BAZEL_EXTRACTED=${be.install_dir}
```

Comment:
Can you pass it through at the nix level instead? You can pass it to the callPackage of `shebang` instead, then it’s less indirect and we drop to the bash level at the last possible moment.

Reply:
Done, PTAL.